app-admin/pinkybar/metadata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+	</maintainer>
+	<maintainer type="person">
+		<email>sacewe@emailure.net</email>
+		<name>Aaron Caffrey</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">su8/pinky-bar</remote-id>
+	</upstream>
+	<use>
+		<flag name="alsa">To get the sound volume level.</flag>
+		<flag name="colours">Colorize the output data</flag>
+		<flag name="cpp">Extend pinkybar with cpp</flag>
+		<flag name="drivetemp">Read the drive temperature from S.M.A.R.T</flag>
+		<flag name="drivetemp-light">Read the drive temperature from S.M.A.R.T light version</flag>
+		<flag name="dvd">To get the cdrom dvdrom vendor and model names</flag>
+		<flag name="libnl">Enable the wifi related options regarding chipsets supporting the cfg80211,mac80211 modules</flag>
+		<flag name="lisp">Extend pinkybar with your own crafted scripts written in lisp, stored in /usr/share/pinkysc/pinky.lisp</flag>
+		<flag name="mpd">To see the currently played song name</flag>
+		<flag name="ncurses">Output the data to the terminal using the ncurses library</flag>
+		<flag name="net">Enable the internet related options</flag>
+		<flag name="pci">To get the NIC vendor and model names</flag>
+		<flag name="perl">Extend pinkybar with your own crafted scripts written in perl, stored in /usr/share/pinkysc/pinky.pl</flag>
+		<flag name="python2">Extend pinkybar with your own crafted scripts written in python, stored in /usr/share/pinkysc/pinky.py</flag>
```

Comment:
Use 'python', not 'python2'. We just don't do that.

Reply:
The program [uses](https://github.com/su8/pinky-bar/blob/master/m4/extend.m4#L114) python2 and python3. I chosed python2 as it's simplier to use and doesn't require to hack the **PYTHONPATH** variable.